### DISCUSSION TITLE
Welcome to novowrap Discussions!

### DISCUSSION THREAD

**Discussion by wpwupingwp** [wpwupingwp/novowrap#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Scientistmkh** [comment]
Hi,
I tried to aseembly the chloroplast genome with novowrap but I got a error message and failed
here is the log history Please help me to find the solution.

13:44:04 INFO     
13:44:04 INFO     Input file: C:\Users\Harichandra\Desktop\FDF_2.fq.gz
13:44:04 INFO     Input file: C:\Users\Harichandra\Desktop\FDF_1.fq.gz
13:44:04 INFO     Minimum genome size: 100000
13:44:04 INFO     Maximum genome size: 200000
13:44:04 INFO     Taxonomy: Ficus deltoidea
13:44:04 INFO     Output folder: C:\Users\Harichandra\Desktop\Output
13:44:05 INFO     Split 10000000 pairs of reads for assembly
13:46:36 DEBUG    C:\Users\Harichandra\Desktop\Output\Temp\FDF_2.fq.10000000 and C:\Users\Harichandra\Desktop\Output\Temp\FDF_2.fq.10000000 are same.
13:48:42 DEBUG    C:\Users\Harichandra\Desktop\Output\Temp\FDF_1.fq.10000000 and C:\Users\Harichandra\Desktop\Output\Temp\FDF_1.fq.10000000 are same.
13:48:42 INFO     Try to get reference of Ficus deltoidea from NCBI Genbank.
13:48:53 DEBUG    Query from NCBI Genbank:	Ficus_deltoidea[Organism] AND refseq[filter] AND (chloroplast[filter] OR plastid[filter])
13:48:55 DEBUG    Query from NCBI Genbank:	Ficus[Organism] AND refseq[filter] AND (chloroplast[filter] OR plastid[filter])
13:49:07 INFO     Got Ficus_NC_066476.gb as reference.
13:49:07 INFO     Use seed as seed.
13:49:07 INFO     Call NOVOPlasty... May need minutes (rarely half an hour)
13:49:12 WARNING  Assembled with seed failed.
13:49:12 INFO     Use whole as seed.
13:49:12 INFO     Call NOVOPlasty... May need minutes (rarely half an hour)
13:49:13 WARNING  Assembled with whole failed.
13:49:13 INFO     Failed with all seeds.
13:49:13 INFO     Bye.

**Reply by wpwupingwp** [reply]
Hi,
You can try several solutions:
1. use more reads (more than 10000000)
2. manually set assembly seed by giving gene names
3. use a reference genbank file rather than let program download from ncbi
4. check if your computer have sufficient memory. For 10000000 reads, at least 8G or 16GB is recommended 

Ping Wu

**Reply by Scientistmkh** [reply]
Hi Sir,
Thank you for your prompt replay.
I used 10,000,000 (more than 10,000,000 in the sense is that 100,000,000 ?)
I used a reference seed file from NCBI which is having 636 bp (It shoud be 500-700bp right?)
I have 8GB RAM

Please help me to rectify this failing I tried with 10 different seed files 20times i got failed message
Thank you

**Reply by wpwupingwp** [reply]
8GB may not be sufficient. If your data is from fresh sample, you can try 5000000 or less reads. 

However it can be other reasons too. Can you send me the data to let me try on my server? Google drive or other net disk is ok.

Also you could try other software (getorganelle) if you have a PC with big memory.
